src/main/resources/edu/cornell/kfs/module/cg/businessobject/datadictionary/Award.xml

```diff
@@ -165,4 +178,33 @@
             </list>
         </property>
     </bean>
-</beans>
\ No newline at end of file
+    
+    <bean id="CuAward-lookupDefinition" parent="Award-lookupDefinition">
+    	<property name="resultFields">
```

Comment:
Instead of overriding the resultFields list completely, you could try using one of the helper field-overriding beans to add the "invoiceLink" column to the list. That should keep this field setup a lot simpler.

Reply:
I want the invoice link field to be the firm column, I think to make the first item, I need to do it lke this

Reply:
If you wanted to simplify this further, you could add some "DataDictionaryBeanOverride" and "FieldOverrideForListElementInsert" beans to handle inserting the "invoiceLink" column. Those beans should allow you to find a field within the "resultFields" list and then insert new beans before or after that field. The cu-kfs project should have several examples of this.